main.py

```diff
@@ -60,6 +63,7 @@ class Properties:
     source_code_repository = "P1324"
     title = "P1476"
     protocol = "P2700"
+    license = "P275"
```

Comment:
Apparently there's a builtin function `license()`. Not sure if that actually matters here though

Reply:
Get's overwritten and is not needed for anything (just shows the license of the python interpreter)